src/main/common/components/payment-intention/payment-plan.njk

```diff
@@ -14,14 +14,6 @@
     <form method="post" novalidate class="payment-plan-calculation">
       {{ csrfProtection(csrf) }}
 
-      {% if monthlyIncomeAmount and monthlyExpensesAmount %}
```

Comment:
Since you removed these lines it makes also sense to track down where these values used to be populated and likely remove populating code as well.